utils/gridlink_impl.c.src

```diff
@@ -55,9 +55,9 @@ void free_cellarray_DOUBLE(cellarray_DOUBLE *lattice, const int64_t totncells)
 
 cellarray_DOUBLE * gridlink_DOUBLE(const int64_t NPART,
                                    DOUBLE *X, DOUBLE *Y, DOUBLE *Z, weight_struct *WEIGHTS,
-                                   const DOUBLE xmin, const DOUBLE xmax,
-                                   const DOUBLE ymin, const DOUBLE ymax,
-                                   const DOUBLE zmin, const DOUBLE zmax,
+                                   const DOUBLE xmin, const DOUBLE xdiff,
+                                   const DOUBLE ymin, const DOUBLE ydiff,
+                                   const DOUBLE zmin, const DOUBLE zdiff,
```

Comment:
Definitely prefer to keep the old signature, if at all possible. Otherwise, add in a separate `xdiff` parameter (or preferably, a differently named `xwrap`)

Reply:
Now changed back to the original signature.